services/net/reporting/ReportingManager.cs

```diff
@@ -1016,17 +1017,21 @@ private async Task<Dictionary<EmailSentTo, List<UserEmail>>> GetEmailAddressesAs
         }
         else
         {
-            switch (sendTo)
+            var user = await this.Api.GetUserAsync(userId);
```

Comment:
This will add a lot of additional requests for each report.  We already do a join on the user table to get all the user's email addresses.  You can update the model to include the vacation mode.